index.html

```diff
@@ -27,8 +27,14 @@
             <li class=" header__nav-container__nav-item"><a class="underline-animation" href="#about">About</a></li>
             <li class=" header__nav-container__nav-item"><a class="underline-animation" href="#contact">Contact</a></li>
         </ul>
-        <nav class="header__nav-container__mobile_menu">
-            <img src="./images/menu.svg" alt="hamburgar menu" />
+        <img id="mobile_menu_open" class="mobile_menu_open" src="./images/menu.svg" alt="hamburgar menu" />
+        <nav id="mobile_nav" class="header__nav-container__mobile_menu hidden">
+            <img id="mobile_menu_close" src="./images/close-icon.svg" alt="mobile menu close icon" />
```

Comment:
Awesome work ✔ The hamburger menu button is visible on mobile size and hides when changed to desktop size 😎